### DISCUSSION TITLE
蓝奏云获取不到下载链接

### DISCUSSION THREAD

**Discussion by xl3366** [AlistGo/alist#618] [Q&A]
{"code":500,"message":"Get \"https:///iWAlPyy3khi\": http: no Host in request URL","data":null}
![image](https://user-images.githubusercontent.com/47490922/155768656-34e7d3ad-c2d0-4c1b-958d-06faa9c46c66.png)

**Comment by xhofe** [comment]
https://alist-doc.nn.ci/docs/driver/lanzou
